deploy-help/deploy.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -e
+
```

Comment:
Please rename the script to `build-and-deploy-to-dev.sh`. Put it in `concent-builder/files/` instead and make `configure.yml` upload it to `/usr/local/bin` on the server along with other scripts.